bingosync-app/bingosync/models.py

```diff
@@ -151,12 +255,36 @@ def choices():
     GameType.pokemon_red_blue_randomizer: "Poké Random",
 }
 
+class LockoutMode(Enum):
+    non_lockout = 1
+    lockout = 2
+
+    def __str__(self):
+        return LOCKOUT_MODE_NAMES[self]
+
+    @staticmethod
+    def for_value(value):
+        return list(LockoutMode)[value - 1]
+
+    @staticmethod
+    def default_value():
+        return LockoutMode.non_lockout.value
+
+    @staticmethod
+    def choices():
+        return [(lockout_mode.value, str(lockout_mode)) for lockout_mode in LockoutMode]
+
+LOCKOUT_MODE_NAMES = {
+    LockoutMode.non_lockout: "Non-Lockout",
```

Comment:
Yeah, I think Non-lockout is a better descriptor, as having it set like that is useful for more than just Blackout